cfme/containers/pod.py

```diff
@@ -8,6 +8,12 @@
 
 list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
 
+
```

Comment:
@psav So basically, we would get to the provider's detail page, click on the Pods field in the Relationships table then define the ui_pods? The function should then return the list of pods. Should it be inside the Provider class?